### DISCUSSION TITLE
Ring Smart Lighting Echo Hub

### DISCUSSION THREAD

**Discussion by thumbsup23** [dgreif/ring#821] [Q&A]
Hello,

Does this support bringing in Ring Smart Lights that are using an Echo 4th Gen as the hub? I have it setup but can't seem to get them to load into HomeBridge. Was just wondering if anything is different because I am using the echo as the hub instead of the ring bridge but inside the app they seem the same.

**Comment by dgreif** [comment]
I wasn't aware that was an option!  I've only tested using the official Ring Smart Lighting Hub.  Do the lights show up in the Ring app when connected via the Echo?

**Reply by thumbsup23** [reply]
Yeah everything is handled from the ring app with setup and usage.

**Reply by thumbsup23** [reply]
![44898174-5A58-448D-B40E-E98A7529F52D_1_102_o](https://user-images.githubusercontent.com/94005264/141482451-a1813eff-2b58-4462-9ef9-7fb53f92999f.jpeg)


Here is what it looks like in the ring app, not sure how it looks with the ring bridge.
